apps/crn-frontend/src/events/api.ts

```diff
@@ -1,14 +1,29 @@
 import { AlgoliaSearchClient } from '@asap-hub/algolia';
-import { EventResponse, ListEventResponse } from '@asap-hub/model';
 import { createSentryHeaders } from '@asap-hub/frontend-utils';
-
+import { EventResponse, ListEventResponse } from '@asap-hub/model';
 import { API_BASE_URL } from '../config';
-import { GetEventListOptions } from './options';
 import createListApiUrl from '../CreateListApiUrl';
+import { GetEventListOptions } from './options';
+
+export function getFilters(filters: string[], userId?: string) {
+  if (filters.length === 0) return undefined;
```

Comment:
This doesn't seem to cover if just a user id is specified which seems odd.

Reply:
This has moved the the algolia package, and now should filter by userId